src/app/components/Input/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import clsx from 'clsx';
+
+import styles from './styles.module.scss';
+
+interface InputProps extends React.HTMLProps<HTMLInputElement> {
```

Comment:
same change interfacename 
```suggestion
interface Props extends React.HTMLProps<HTMLInputElement> {
```